ci/jenkins/cluster-templates/cluster.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: cluster.x-k8s.io/v1alpha2
```

Comment:
I noticed that there is a new release (`cluster.x-k8s.io/v1alpha3`), can we use that instead?

See https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/blob/master/examples/default/cluster/cluster.yaml#L3

Reply:
You are right. It is better to move them out of the jenkins path.

I prefer not to change `v1alpha2` to `v1alpha3` for 2 reasons:
* The CAPV doc is still using `v1alpha2` as an example in their get_started page. Other developers are likely to follow their instruction and end up getting error when using our code.
* Our current management cluster is not compatible with `v1alpha3` and we already have current management cluster setup code in ansible playbook.

I think we can move to `v1alpha3` when CAPV officially recommends using that. The CAPV code already has some other problems and I think it is better to stay at a stable version.

Reply:
sounds good